yt_dlp/extractor/arte.py

```diff
@@ -152,22 +153,26 @@ def _real_extract(self, url):
                     not m.group('sdh_sub'),                 # and we prefer not the hard-of-hearing subtitles if there are subtitles
                 )))
 
+            short_label = traverse_obj(stream_version, "shortLabel", expected_type=str, default='?')
```

Comment:
```suggestion
            short_label = traverse_obj(stream_version, 'shortLabel', expected_type=str, default='?')
```

Reply:
Right, that was copied from the old source, which uses double quotes.